sound/soc/sof/sof-audio.h

```diff
@@ -282,6 +282,16 @@ struct sof_token_info {
 	int count;
 };
 
+/**
+ * struct snd_sof_pipeline_trigger_info - multi pipeline trigger IPC info
```

Comment:
How this struct is 'multi pipeline trigger **IPC info**'?
It is more like pipeline path or pipeline list.

In any case, I find it increasingly confusing that `snd_sof_widget` is sometimes a widget, module, component or a pipeline.

Probably it is really a snd_sof_pipeline_trigger_path? Certainly not IPC info.

Reply:
And this does not necessarily needs to be valid for paths which consists of more than one pipeline (we always have at least 2 pipelines with IPC4, are they also multi-pipeline?)